packages/api/endpoints/providers.js

```diff
@@ -53,18 +54,21 @@ async function list(req, res) {
  */
 async function get(req, res) {
   const id = req.params.id;
+  const knex = await getKnexClient({ env: process.env });
+  const providerPgModel = new ProviderPgModel();
 
-  const providerModel = new Provider();
   let result;
   try {
-    result = await providerModel.get({ id });
+    const providerRecord = await providerPgModel.get(knex, { name: id });
+    result = translatePostgresProviderToApiProvider(providerRecord);
   } catch (error) {
-    if (error instanceof RecordDoesNotExist) return res.boom.notFound('Provider not found.');
+    if (error instanceof RecordDoesNotExist) return res.boom.notFound(`Provider ${JSON.stringify(id)} not found.`);
+    throw error;
   }
-  delete result.password;
-  return res.send(result);
+  return res.send(removeNilProperties(result));
```

Comment:
This is redundant, and could probably be removed.

Reply:
Why do we want to remove the null properties? Is that consistent with what we typically return for this endpoint?

Reply:
It is - and in fact *not* removing the null properties breaks some of our internal code.   We should be doing it for our other endpoints I believe.